v0/destinations/af/transform.js

```diff
@@ -207,12 +207,17 @@ function processSingleMessage(message, destination) {
       break;
     }
     case EventType.SCREEN: {
-      const eventName = EventType.SCREEN;
+      const eventName = `Viewed ${message.name ||
+        message.event ||
+        get(message, "properties.category") ||
+        ""} Screen`;
```

Comment:
Updating event name might impact existing customers

Reply:
Agree with you, so provided a switch on the dashboard for the user to enable the screen/page name in the screen/page call.